client.py

```diff
@@ -105,21 +114,20 @@ def configure():
     f.write("USER_ID = " + str(user_id))
     f.close()
 
-
-if len(sys.argv) < 2:
-    print "Usage: ./client.py <option>"
-    print "Options:"
-    print "-c, --configure => Run first-time configuration"
-    print "-e, --encrypt => Run the prePush sequence"
-    print "-d, --decrypt => Run the postPull sequence"
-    sys.exit(1)
-
-if sys.argv[1] in ['-c', '--configure']:
-    configure()
-    print "Configuration successful!"
-elif sys.argv[1] in ['-e', '--encrypt']:
-    prePush(raw_input("Enter file path: "))
-elif sys.argv[1] in ['-d', '--decrypt']:
-    url = raw_input("Enter Dropbox share link: ")
-    originalMessage = postPull(url)
-    print originalMessage
+def main(docopt_args):
+    if docopt_args['--configure']:
+        configure()
+        print "Configuration successful!"
+    elif docopt_args['--encrypt']:
+        prePush(raw_input("Enter file path: "))
+    elif docopt_args['--decrypt']:
+        url = raw_input("Enter Dropbox share link: ")
+        originalMessage = postPull(url)
```

Comment:
Hence the indentation levels have been shifted by one....so i guess Git sees that as a modification.